builtin/source/window.ts

```diff
@@ -0,0 +1,87 @@
+import * as fn from "@denops/std/function";
+
+import { defineSource, type Source } from "../../source.ts";
+
+type Detail = {
+  /**
+   * Window ID
+   */
+  winid: number;
+
+  /**
+   * Window number
+   */
+  winnr: number;
+
+  /**
+   * Tab number
+   */
+  tabnr: number;
+
+  /**
+   * Buffer number in the window
+   */
+  bufnr: number;
+
+  /**
+   * Buffer name in the window
+   */
+  bufname: string;
+
+  /**
+   * Window information
+   */
+  wininfo: fn.WinInfo;
```

Comment:
Do we really need this? It seems all information in `wininfo` is already expanded to the `Detail` itself.